src/applications/caregivers/definitions/caregiverUI.js

```diff
@@ -102,8 +108,19 @@ export default {
       'ui:title': ' ',
       'ui:order': ['name', 'type'],
       name: {
-        'ui:title':
-          'Please enter the name of the medical facility where the Veteran last received medical treatment.',
+        'ui:title': (
+          <div>
+            <h3 className="vads-u-font-size--h4">Recent medical care</h3>
+            <p>
+              Please enter the name of the medical facility where the Veteran
+              <b className="vads-u-margin-left--0p5">
+                last received medical treatment.
+              </b>
```

Comment:
`<strong>` is the recommended element to mark bold text.

Reply:
Change all `b` tags to `strong` tags across app